src/prelude.rs

```diff
@@ -10,8 +10,12 @@ pub use alloc::{
     vec,
     vec::Vec,
 };
+#[cfg(all(feature = "testnet", not(feature = "std")))]
+pub use core::ops::Add;
```

Comment:
Possibly not needed? (See above comment)

As a side note, is it necessary to distinguish the std and no-std cases? By which I mean if I import `core::ops::Add` when std is also used in the library will that fail to compile?

Reply:
Yep, not needed. Good eye.